stingray/tests/test_utils.py

```diff
@@ -52,6 +52,15 @@ def test_uneven_binned_counts(self):
         ybin_test = np.zeros_like(xbin) + self.counts*dx_new/self.dx
         assert np.allclose(ybin_test, ybin)
 
+    def test_rebin_data_should_raises_error_when_method_is_different_than_allowed(self):
+        dx_new = 2.0
+        try:
+            utils.rebin_data(self.x, self.y, dx_new, method='not_allowed_method')
+        except utils.UnrecognizedMethod:
+            pass
+        else:
+            raise AssertionError("Expected exception does not occurred")
+
```

Comment:
Based on #89 the syntax here should now be:

    with pytest.raises(utils.UnrecognizedMethod):
        utils.rebin_data(self.x, self.y, dx_new, method='not_allowed_method')

(Or just `pytest.raises(UnrecognizedMehod)`? I'm not sure how to catch a custom exception)